setup.py

```diff
@@ -0,0 +1,31 @@
+from setuptools import setup
+import os.path
+import sys
+
+setup(
+    name = 'sphinx_fossasia_theme',
+    version = '0.0.1',
+    author = 'Ujjwal Bhardwaj',
+    author_email = 'ujjwalb1996@gmail.com',
+    url = 'https://github.com/fossasia/sphinx_fossasia_theme',
+    license = 'GNU',
+    description = 'A Sphinx theme specific to FOSSASIA\'s Projects',
+    packages = ['sphinx_fossasia_theme'],
+    include_package_data = True,
+    entry_points = {
+        'sphinx.html_themes': [
```

Comment:
Confirmed that this does not work for sphinx versions < 1.6

Use the below snippet to get the correct entry point depending on the sphinx version
```python
sphinx_entry_point = 'sphinx_themes' if sphinx.version_info[:2] < (1, 6) else 'sphinx.html_themes'
```